### PR TITLE
Arrays

### PR DESCRIPTION
**Pull request by OlgaRomankova**

### CODE REVIEW

File: Arrays.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+// [ 'cat, 'dog', 'raccon' ] => [ 'cat', 'dog', 'racoon' ]
+// */
+// function getArrayOfStrings(arr) {
+// return
+// }
+// /**
+
+function getArrayOfStrings(arr) {
+  
+      return arr.filter(funcСheckString)
```
Comment (GarinaKristina) [inline Arrays.js]: I didn`t catch what return this function

File: Arrays.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+// */
+// function getUpperCaseStrings(arr) {
+// return
+// }
+// /**
+
+
+function getUpperCaseStrings(arr) {
+   const upper = arr.map(element => {
+      return element.toUpperCase();
```
Comment (GarinaKristina) [inline Arrays.js]: Please, check the logic this function

Comment (OlgaRomankova) [reply Arrays.js]: Thank you, the problem is solved

**Comment by GarinaKristina** [comment]
Tasks for new branch :
/**
 * Transforms the numeric array into the according array of squares:
 *   f(x) = x * x
 *
 * @param {array} arr
 * @return {array}
 *
 * @example
 *   [ 0, 1, 2, 3, 4, 5 ] => [ 0, 1, 4, 9, 16, 25 ]
 *   [ 10, 100, -1 ]      => [ 100, 10000, 1 ]
 */
function toArrayOfSquares(arr) {
  return 
}

/**
 * Returns the 3 largest numbers from the specified array
 *
 * @param {array} arr
 * @return {array}
 *
 * @example
 *   [] => []
 *   [ 1, 2 ] => [ 2, 1 ]
 *   [ 1, 2, 3 ] => [ 3, 2, 1 ]
 *   [ 1,2,3,4,5,6,7,8,9,10 ] => [ 10, 9, 8 ]
 *   [ 10, 10, 10, 10 ] => [ 10, 10, 10 ]
 */
function get3TopItems(arr) {
  return 


/**
 * Sorts the specified array by country name first and city name (if countries are equal) in ascending order.
 *
 * @param {array} arr
 * @return {array}
 *
 * @example
 *    [
 *      { country: 'Russia',  city: 'Moscow' },
 *      { country: 'Belarus', city: 'Minsk' },
 *      { country: 'Poland',  city: 'Warsaw' },
 *      { country: 'Russia',  city: 'Saint Petersburg' },
 *      { country: 'Poland',  city: 'Krakow' },
 *      { country: 'Belarus', city: 'Brest' }
 *    ]
 *                      =>
 *    [
 *      { country: 'Belarus', city: 'Brest' },
 *      { country: 'Belarus', city: 'Minsk' },
 *      { country: 'Poland',  city: 'Krakow' },
 *      { country: 'Poland',  city: 'Warsaw' },
 *      { country: 'Russia',  city: 'Moscow' },
 *      { country: 'Russia',  city: 'Saint Petersburg' }
 */
//=======================================================================================================================================
function sortCitiesArray(arr) {
  return 
}

**Comment by GarinaKristina** [comment]
https://www.codewars.com/kata/544675c6f971f7399a000e79

**Comment by GarinaKristina** [comment]
https://www.codewars.com/kata/56bc28ad5bdaeb48760009b0

File: Arrays.js (hunk 1 above)
Comment (OlgaRomankova) [reply Arrays.js]: upd

File: Arrays.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+// [ '', 'a', 'bc', 'def', 'ghij' ] => [ 0, 1, 2, 3, 4 ]
+// [ 'angular', 'react', 'ember' ] => [ 7, 5, 5 ]
+// */
+// function getStringsLength(arr) {
+// return
+// }
+
+ function getStringsLength(arr) {
+   const newArray = arr.map((word) => {return word.length})
+   console.log(newArray)
```
Comment (GarinaKristina) [inline Arrays.js:132]: Good job, but you also could write like this 
`function getStringsLength(arr) {
  return arr.map(word => word.length);
}`

File: Garina tasks third part.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+// isString('test') => true
+// isString(new String('test')) => true
+// */
+// function isString(value) {
+// }
+
+function isString(value) {
+       
+    if (typeof value === 'string' || value instanceof String) {
+       return ("true")} else {return ("false")}
```
Comment (GarinaKristina) [inline Garina tasks third part.js:117]: Good job, just small comment. Following code rules , clean code and others, if else construction must look like this

Screenshot 2024-04-02 at 12 38 45

**Review by GarinaKristina — APPROVED** [review]

File: Arrays.js (hunk 3 above)
Comment (OlgaRomankova) [reply Arrays.js:132]: Ok, thanks

File: Garina tasks third part.js (hunk 4 above)
Comment (OlgaRomankova) [reply Garina tasks third part.js:117]: Thanks, I'll fix the code)

**Pull request merged**